main.go

```diff
@@ -137,25 +139,24 @@ func main() {
 	app.Run(os.Args)
 }
 
-func createServicesMap(services string, messageTypeMap map[string]string, vulcanAddr string) (map[string]string){
+func createServicesMap(services string, messageTypeMap map[string]string, vulcanAddr string) map[string]string {
 	stringSlice := strings.Split(services, ",")
 	servicesMap := make(map[string]string)
 	for _, service := range stringSlice {
 		for messageType, concept := range messageTypeMap {
 			if strings.Contains(service, concept) {
-				//TODO hardcoded url?
-				writerUrl := vulcanAddr + "/__" + service + "/" + concept
-				servicesMap[messageType] = writerUrl
-				fmt.Printf("Added url %v to map:\n", writerUrl)
+				writerURL := vulcanAddr + "/__" + service + "/" + concept
+				servicesMap[messageType] = writerURL
+				fmt.Printf("Added url %v to map:\n", writerURL)
```

Comment:
I think that was just one of my debugging messages that I missed on clean up. Dont really think it needs to stay in the app now.